README.md

```diff
@@ -16,7 +16,7 @@ from configured targets at given intervals, evaluates rule expressions,
 displays the results, and can trigger alerts if some condition is observed
 to be true.
 
-Prometheus' main distinguishing features as compared to other monitoring systems are:
+Prometheus main distinguishing features as compared to other monitoring systems are:
```

Comment:
Prometheus needs to be possessive here, `Prometheus's` would be grammatically correct.

Reply:
Fixed